data/docs/userguide/collect_systemd_logs.mdx

```diff
@@ -0,0 +1,403 @@
+---
+date: 2025-06-25
+title: Collecting SystemD Logs (JournalD) 
+id: collect_systemd_logs
+hide_table_of_contents: true
+---
+
+## Overview
+
+This documentation provides detailed instructions on configuring the OpenTelemetry Collector to read logs from SystemD's journal (JournalD) and push them to SigNoz. SystemD logs are structured and contain rich metadata that can help you monitor system services, troubleshoot issues, and track service performance.
+
+## Prerequisites
+
+Before proceeding, ensure that:
+- SystemD and JournalD are installed and running on your system
+- The `journalctl` binary is available in the `$PATH`
+- You have sufficient permissions to access the journal
+
+## Sample SystemD Logs
+
+SystemD generates structured logs for all services managed by it. Here's what typical journal entries look like:
+
+```plaintext
+Jun 25 10:30:45 hostname systemd[1]: Started myapp.service.
+Jun 25 10:30:46 hostname myapp[1234]: Application started successfully
+Jun 25 10:30:47 hostname myapp[1234]: Processing user request
+```
+
```

Comment:
Its good to add the documentation on how to access the journalctl logs here.

`sudo journalctl -n 10`